blockly-core/core/utils/xml.js

```diff
@@ -17,6 +17,12 @@
  * limitations under the License.
  */
 
+/* global Blockly, goog */
+
+// Ignore "Expected a conditional expression and instead saw an
+// assignment" warnings.
+/* jshint -W084 */
```

Comment:
I don't think blockly-core is actually linted by anything except the new pre-commit hook.